app/Main.hs

```diff
@@ -108,6 +112,11 @@ optionsParser env = Options
        <*> switch
            (  long "no-inherit-env"
            <> help "don't merge the parent environment with the secrets file")
+       <*> (MilliSeconds <$> option auto
+               (  long "retry-base-delay-milliseconds"
+               <> metavar "MILLISECONDS"
+               <> value (40 :: Int)
+               <> help "base delay for vault connection retrying. Defaults to 40ms because, in testing, we found out that fetching 50 secrets takes roughly 200 milliseconds"))
```

Comment:
Could you also update the [usage section in the README](https://github.com/channable/vaultenv#usage) with the changed output? (Some other options might also be missing)